### PR TITLE
pytx Reactions and a few bug fixes

### LINKED ISSUE #137: [pytx] Added privacy_type to Malware objects
Privacy_type is missing on pytx.Malware objects and in pytx.vocabulary.Malware, despite being a field in TX for Malware Analyses

### PR DESCRIPTION
**Pull request by mgoffin**
You can now use `.react()` on a ThreatDescriptor object to provide a
reaction. Vocabulary has been added so you can also provide a reaction
through all of the other ways you can update an object.

Several module attributes (including __version__) have been added.

Fixing bug with Malware where there was no privacy vocabulary. Fixing a
bug with ThreatTag where there was no objects vocabulary.

Bumping version.

This should close #137 and #138.

### CODE REVIEW

File: pytx/pytx/vocabulary.py [hunk 1]
```diff
@@ -194,6 +198,7 @@ class MalwareFamilies(object):
     ID = Common.ID
     MALICIOUS = 'malicious'
     NAME = 'name'
+    PRIVACY_MEMBERS = 'privacy_members'
```
Comment (hammem) [inline pytx/pytx/vocabulary.py]: This field isn't supported on MalwareFamilies just yet.

File: pytx/pytx/vocabulary.py [hunk 2]
```diff
@@ -128,6 +130,8 @@ class Malware(object):
     METADATA = Common.METADATA
     PASSWORD = 'password'
     PE_RICH_HEADER = 'pe_rich_header'
+    PRIVACY_MEMBERS = 'privacy_members'
```
Comment (hammem) [inline pytx/pytx/vocabulary.py]: This field isn't supported on Malware just yet, let's remove for now.

File: pytx/pytx/vocabulary.py [hunk 3]
```diff
@@ -220,6 +225,19 @@ class PagingCursor(object):
...
+    Vocabulary for describing a reaction.
+    """
+
+    HELPFUL = 'HELPFUL'
+    NOT_HELPFUL = 'NOT_HELPFUL'
+    WANT_MORE_INFO = 'WANT_MORE_INFO'
+    SAW_THIS_TOO = 'SAW_THIS_TOO'
+    OUTDATED = 'OUTDATED'
+
+
```
Comment (hammem) [inline pytx/pytx/vocabulary.py]: nit: alphabetize

**Comment by hammem** [comment]
Thanks, @mgoffin !  There are a few minor details inline.

File: pytx/pytx/vocabulary.py (hunk 3 above)
Comment (bryongloden) [inline pytx/pytx/vocabulary.py]: Greetings @hammem! Was that 'nit' like nitpicking?
Greetings @mgoffin! Just alphabetize this one in descending order (fan favorite) 👍

**Comment by mgoffin** [comment]
Thanks for the head's up on those fields and correcting the alphabetizing. And thanks Travis for noticing I didn't fix something I should have :)

**Comment by hammem** [comment]
Thanks, @mgoffin !

File: pytx/pytx/vocabulary.py (hunk 3 above)
Comment (hammem) [inline pytx/pytx/vocabulary.py]: @bryongloden, yep 'nit' => nitpicking

**Pull request merged**